pages/docs/architecture.md

```diff
@@ -129,68 +129,68 @@ To help getting started with containers on embedded systems, resinOS comes with
 Debian, Fedora and Alpine Linux distributions, as well as Nodejs, Python, Go and Java language base images. For a more in-depth look into all the available base images head over
 to the [resin.io base images wiki](http://docs.resin.io/runtime/resin-base-images/) or the [resin dockerhub repository](https://hub.docker.com/u/resin/).
 
-### Resin Device Toolbox
+### Resin Command Line Tool
 
-The Resin Device Toolbox or `rdt` for short, is a set of useful tools that help with setting up and developing containers with a resinOS device. The goal of `rdt` is to provide
-a simple and intuitive developer experience. Currently, `rdt` is evolving fast and is considered 'alpha', so as always, we love it when you report bugs,
-you can report them here: [github.com/resin-os/resin-device-toolbox](https://github.com/resin-os/resin-device-toolbox/issues)
+The resin CLI is a set of useful tools that help with setting up and developing containers with a resinOS device. The goal of the CLI is to provide
+a simple and intuitive developer experience. Currently, local mode part of the CLI are evolving fast and is considered 'beta', so as always, we love it when you report bugs,
+you can report them here: [github.com/resin-io/resin-cli](https://github.com/resin-io/resin-cli)
 
 #### Installation
-Currently `rdt` is a node.js based command line tool which requires that our system has the following dependencies installed and in our path:
+Currently the CLI is a node.js based command line tool which requires that our system has the following dependencies installed and in our path:
 
 * [node.js 6.x](https://nodejs.org/en/)
 * [npm package manager](https://www.npmjs.com/)
 * [rsync](https://download.samba.org/pub/rsync/rsync.html)
 * [ssh](http://www.openssh.com/)
 
-Once we have those setup we can install `rdt` using npm:
+Once we have those setup we can install the CLI using npm:
 ``` bash
-$ npm install -g resin-device-toolbox
+$ npm install --global --production resin-cli
 ```
 #### Usage
 ##### Configure
-`rdt configure` allows you configure or reconfigure a resinOS system image or SD card. Currently, this allows for configuration of wifi settings, hostname and enablement of
+`resin local configure` allows you configure or reconfigure a resinOS system image or SD card. Currently, this allows for configuration of wifi settings, hostname and enablement of
```

Comment:
This feels like it'll get out of date quick. Can we not just link pull from resin cli docs and drop in a partial?